INSTALLATION.md

```diff
@@ -0,0 +1,164 @@
+## Pre-built wheels
```

Comment:
I think this is useful but should either be in README.md or in the docs (https://pytorch.org/TensorRT/getting_started/installation.html) and we link to it in the readme.

Reply:
I'd like to keep the README as lightweight as possible, I will port this content over to the docs (if any changes are required)